packages/client/hmi-client/src/composables/useDrilldownChartSize.ts

```diff
@@ -12,9 +12,15 @@ import { useResizeObserver } from './useResizeObserver';
  */
 export function useDrilldownChartSize(containerElement: Ref<HTMLElement | null>, debounceDelay = 100) {
 	const size = ref(drilldownChartSize(null));
+	watch(containerElement, (val) => {
+		// Init the size as soon as the container element is available
+		if (!val) return;
+		size.value = drilldownChartSize(containerElement.value);
+	});
```

Comment:
hmm ... is this going to create a memory leak every time we invoke the composable? Who is responsible for stopping the watcher?

Reply:
May be relevant: https://vuejs.org/guide/essentials/watchers#stopping-a-watcher

Reply:
Looks like it will be cleared when the owner component is destroyed